frontend/src/lib/components/PropertyFilters/PropertyFilters.tsx

```diff
@@ -23,6 +23,7 @@ interface PropertyFiltersProps {
     taxonomicGroupTypes?: TaxonomicFilterGroupType[]
     showNestedArrow?: boolean
     addButtonType?: ButtonType
+    greyBadges?: boolean
```

Comment:
non-blocking

Does `greyBadges` have a more semantic naming option? Or is it literally just that it should be grey? e.g. `subtleBadges` or something similar.

Reply:
So this was prompted by Chris' comment above to update the styling. Rather than update the whole app I decided to do the minimum thing with the expectation this will get cleared up/removed with the upcoming insights style changes.

I like `greyBadges` for a short-term name over `subtle` because less context :)